### ISSUE TITLE
UAA must wait for the database to be ready before starting

### ISSUE THREAD

**Issue by f0rmiga** [SUSE/scf#2975]
Sometimes the UAA starts before the database is fully ready. When this happens, UAA never gets ready. A similar approach for waiting on the database exists on SCF v2.x.

**Comment by loewenstein** [comment]
@f0rmiga Would that be solved by https://github.com/cloudfoundry-incubator/cf-operator/issues/636?

**Comment by c0d1ngm0nk3y** [comment]
> @f0rmiga Would that be solved by [cloudfoundry-incubator/cf-operator#636](https://github.com/cloudfoundry-incubator/cf-operator/issues/636)?

Yes, then `uaa` will wait for `diego-api` which itself will wait for `database`. So it cannot occur anymore.

**Comment by f0rmiga** [comment]
Oh great! It sounds like an elegant solution for this problem.
